.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,106 @@
+name: Report a bug
+description: Errors and regression reports with complete reproducing test cases.
+labels: ["status:triage", "bug"]
```

Ignore my comment here if we're changing the current label "needs triage".
```suggestion
labels: ["needs triage", "bug"]
```

---

Modified